tests/test_prepro_labels.py

```diff
@@ -0,0 +1,49 @@
+# the inclusion of the tests module is not meant to offer best practices for
```

Comment:
Great to see that you started the testing! :-)
I would highly recommend to use ``pytest``. The only minor disadvantage is that you need to add a new package ... but nothing else!

Reply:
Thanks, using pytest now